services-matching/src/test/java/com/mapbox/api/matching/v5/MapboxMapMatchingTest.java

```diff
@@ -699,6 +699,28 @@ public void testWithWaypointNames() throws Exception {
     assertEquals("Ok", response.body().code());
   }
 
+  @Test
+  public void testIgnore() throws Exception {
+    MapboxMapMatching mapMatching = MapboxMapMatching.builder()
+      .profile(PROFILE_DRIVING)
+      .coordinates(Arrays.asList(
+          Point.fromLngLat(2.344003915786743,48.85805170891599),
+          Point.fromLngLat(2.346750497817993,48.85727523615161)))
+      .addIgnore(MapboxMapMatching.IGNORE_ONEWAYS, MapboxMapMatching.IGNORE_ACCESS)
```

Comment:
Could we add `IGNORE_RESTRICTIONS` as well to avoid developer regressions? Refs. above typo's comment.